.github/workflows/historization.yml

```diff
@@ -0,0 +1,74 @@
+name: Historization
+
+on:
+  workflow_dispatch:
+  #schedule:
+  #  - cron: '*/30 * * * *'
```

Comment:
This is just temporarily commented out, right?

Reply:
Yes, until it is proven working.

In the long run, we will go for a sligthly different approach anyway. Currently, the workflow reconstructs all(!) the historical versions from the git history in every(!) run. What we actually want is to only add the newest version to the `daten` repository, but that change will only come next after this PR is working as expected.